HISTORY.md

```diff
@@ -22,6 +22,7 @@
 
 ### Bug Fixes
 * Fail recovery and report once hitting a physical log record checksum mismatch, while reading MANIFEST. RocksDB should not continue processing the MANIFEST any further.
+* Fix a bug in PartitionedIndexBuilder to update FlushPolicy to point to internal key partitioner when it changes from user-key mode to internal-key mode in index partition.
```

Comment:
For release note it's nice to describe in terms of affected user APIs / options, so people can quickly check whether they're affected (in this case the affected option is `index_type == kTwoLevelIndexSearch`)